orchestrator/client/src/main.rs

```diff
@@ -94,7 +93,7 @@ lazy_static! {
     "Usage:
         {} cosmos-to-eth --cosmos-phrase=<key> --cosmos-grpc=<url> --cosmos-prefix=<prefix> --cosmos-denom=<denom> --amount=<amount> --eth-destination=<dest> [--no-batch] [--times=<number>]
         {} eth-to-cosmos --ethereum-key=<key> --ethereum-rpc=<url> --cosmos-prefix=<prefix> --contract-address=<addr> --erc20-address=<addr> --amount=<amount> --cosmos-destination=<dest> [--times=<number>]
-        {} deploy-erc20-representation --cosmos-grpc=<url> --cosmos-prefix=<prefix> --cosmos-denom=<denom> --ethereum-key=<key> --ethereum-rpc=<url> --contract-address=<addr> --erc20-name=<name> --erc20-symbol=<symbol> --erc20-decimals=<decimals>
+        {} deploy-erc20-representation --cosmos-grpc=<url> --cosmos-prefix=<prefix> --cosmos-denom=<denom> --ethereum-key=<key> --ethereum-rpc=<url> --contract-address=<addr>
```

Comment:
This is the main thing I wanted to achieve on this PR; Allow the user to run deploy-erc20-representation with whatever --cosmos-denom they happen to know / remember (eg: somm or usomm) and expect their erc-20 to get deployed sensibly.

Reply:
Yes! Great UX improvement. But I think we should state that it works by display denom or base denom and nothing else.